Container Security/kernel-eks-implementation-guide.md

```diff
@@ -0,0 +1,242 @@
+   and Falcon Container Downloads to install this helm chart and have it work
+   correctly as a specialized container has to exist in the container registry
+   before this chart will install properly.
+
+   The CrowdStrike Falcon sensor should be spinning up on all your Kubernetes nodes
+   now. There should be no further action on your part unless no Falcon Sensor
+   container exists in your registry. If you forgot to add a Falcon Sensor image to
+   your image registry before you ran `helm install`, please add the Falcon Sensor
+   now; otherwise, pods will fail with errors and crash until there is a valid
+   image to pull. The default image name to deploy a kernel sensor to a node is
+   `falcon-node-sensor`.
+   ```
+   To learn more about falcon-helm visit [upstream github](https://github.com/CrowdStrike/falcon-helm).
+
+ - (optional) Verify that falcon-node-sensor pod running. You should see one pod per each node on your cluster.
+   ```
+   $ kubectl get pods -n falcon-system
+   NAME                              READY   STATUS    RESTARTS   AGE
+   falcon-helm-falcon-sensor-bs98m   2/2     Running   0          21s
+   ```
+ - (optional) Verify that given pod has registered with CrowdStrike Falcon and received unique identifier.
+   ```
+   $ kubectl exec -n falcon-system falcon-helm-falcon-sensor-bs98m -c falcon-node-sensor -- falconctl -g --aid
+   aid="a582XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+   ```
+ - (optional) Verify that Falcon Sensor for Linux has insert itself to the kernel
+    ```
+    $ lsmod | grep falcon
+    falcon_lsm_serviceable     724992  1
+    falcon_nf_netcontain        20480  1
+    falcon_kal                  45056  1 falcon_lsm_serviceable
+    falcon_lsm_pinned_11110     45056  1
+    falcon_lsm_pinned_11308     45056  1
+    ```
+
+## Uninstall helm chart
+
+ - Step 1: Uninstall the helm chart
+   ```
+   helm uninstall falcon-helm -n falcon-system
+   ```
+ - Step 2: Delete the falcon image from AWS ECR registry
+   ```
+   $ aws ecr batch-delete-image --region eu-west-1 \
+       --repository-name falcon-node-sensor \
+       --image-ids imageTag=latest
+   {
+       "imageIds": [
+           {
+               "imageDigest": "sha256:e14904d6fd47a8395304cd33a0d650c2b924f1241f0b3561ece8a515c87036df",
+               "imageTag": "latest"
+           }
+       ],
+       "failures": []
+   }
+   ```
+ - Step 4: Delete the AWS ECR repository
+   ```
+   $ aws ecr delete-repository --region eu-west-1 --repository-name falcon-node-sensor
+   {
+       "repository": {
+           "repositoryArn": "arn:aws:ecr:eu-west-1:123456789123:repository/falcon-sensor",
+           "registryId": "123456789123",
+           "repositoryName": "falcon-sensor",
+           "repositoryUri": "123456789123.dkr.ecr.eu-west-1.amazonaws.com/falcon-sensor",
+           "createdAt": "2021-02-04T10:30:30+00:00",
+           "imageTagMutability": "MUTABLE"
+       }
+   }
+   ```
+ - Step 5: Delete the AWS EKS Fargate Cluster
```

Comment:
```suggestion
 - Step 5: Delete the AWS EKS Fargate Cluster, replacing `eu-west-1` with the AWS region your cluster is running on.
```

Reply:
I think some of the minor corrections Shawn has identified can be committed. Could we merge this? I will walk through the instructions from the main branch here and add/remove on a fork. Trying to get these out to the customer.

Reply:
Thank You @mccbryan3 and @shawndwells!